lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -742,6 +743,22 @@ public void removeAllAnimatorListeners() {
     animator.removeAllListeners();
   }
 
+  public void addAnimatorPauseListener(Animator.AnimatorPauseListener listener) {
```

Comment:
I think it's safer to add `@RequiresApi(api = Build.VERSION_CODES.KITKAT)` to the public API here. Most people don't pay attention to the logs.

Reply:
I agree, should I remove the `if`statement then?

Reply:
@heitorcolangelo Oh you're right. I hadn't had my coffee yet!